dask_kubernetes/operator/tests/test_customresources.py

```diff
@@ -1,19 +1,20 @@
-import pytest
-
-from glob import glob
-import pathlib
-import os
-
-DIR = pathlib.Path(__file__).parent.absolute()
-
-
-@pytest.fixture(scope="session", autouse=True)
-def customresources(k8s_cluster):
-    crd_path = glob(os.path.join(DIR, "..", "customresources", "*"))
-    k8s_cluster.kubectl("apply", "-f", *crd_path)
-    yield
-    k8s_cluster.kubectl("delete", "-f", *crd_path)
-
-
-def test_customresources(k8s_cluster):
-    assert "daskclusters.kubernetes.dask.org" in k8s_cluster.kubectl("get", "crd")
+import pytest
+
+from glob import glob
+import pathlib
+import os

```

Comment:
This diff looks a bit strange given these lines haven't changed.

I suspect this is Windows CRLF vs *nix LF shenanigans. You probably need to tweak your git settings to fix this. [This blog post looks useful.](https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/)